### ISSUE TITLE
Output of `interleav_wrap_chat`

### ISSUE THREAD

**Issue by wlin-at** [InternLM/InternLM-XComposer#301]
Hi, thanks for the great work. I tried the following code snippet with the `internlm-xcomposer2-vl-7b` model for QA task with two input images. 
```
images = [osp.join( image_folder_dir, "COCO_val2014_000000143961.jpg"),
          osp.join( image_folder_dir, "COCO_val2014_000000274538.jpg")]
image1 = model.encode_img(images[0])
image2 = model.encode_img(images[1])
image = torch.cat((image1, image2), dim=0)
query = """First picture:<ImageHere>, second picture:<ImageHere>. Describe the subject of these two pictures?"""
response, _ = model.interleav_wrap_chat(tokenizer, query, image, history=[], meta_instruction= True)
```
(here the `meta_instruction` is a required positional argument, not sure whether it should be set to True or False)
However, I realized that the returned `response` is actually `{'inputs_embeds': wrap_embeds}`. 
How should I further proceed to get the decoded text output? 
Thanks in advance!

**Comment by HoraceXIaoyiBao** [comment]
with torch.cuda.amp.autocast():
            with torch.no_grad():
              image1 = model.encode_img(i["image"][0])
              image2 = model.encode_img(i["image"][1])
              image = torch.cat((image1, image2), dim=0)
      
              response, _ = model.interleav_wrap_chat(tokenizer, text, image, history=[], meta_instruction= True)
              
              
              output = model.generate(
                          inputs_embeds=response["inputs_embeds"].half(),
                          streamer=None,
                          max_new_tokens=128,
                          do_sample=False,
                          temperature=1.0,
                          top_p=0.8,
                          eos_token_id=eos_token_id,
                          repetition_penalty=1.005,
                      )
           
              output = output[0].cpu().tolist()
              response = tokenizer.decode(output, skip_special_tokens=True)

I write this and that works for me, you can have a try.

**Comment by yuhangzang** [comment]
Thanks to @HoraceXIaoyiBao for providing the solution. Our recent [IXC 2.5](https://huggingface.co/internlm/internlm-xcomposer2d5-7b) provides the official examples for multi-image inputs.
